thirdeye/thirdeye-frontend/app/pods/manage/alerts/template.hbs

```diff
@@ -0,0 +1,144 @@
+<aside class="manage-alert-filters col-md-3">
```

Comment:
I'm fine as long as the layout line up. I'm anticipating that we will need to use rows at some point, and the elements inside of it will not line up with the navbar and/or the rest of the page. 
I feel that always adding a row in container is easier than adding special  css classes to offset the container's margin.